pkg/api/mlflow/service/run/service.go

```diff
@@ -520,7 +520,11 @@ func (s Service) LogParam(
 
 	param := convertors.ConvertLogParamRequestToDBModel(run.ID, req)
 	if err := s.paramRepository.CreateBatch(ctx, 1, []models.Param{*param}); err != nil {
-		return api.NewInternalError("unable to insert params for run '%s': %s", run.ID, err)
+		if _, ok := err.(repositories.ParamConflictError); ok {
```

Comment:
It seems like it would be nice to have unit tests demonstrating the different error cases.

Reply:
I think we do cover this in service_test.go

Reply:
i can add same test for LogBatch..